test/examples_test.go

```diff
@@ -15,25 +15,45 @@ func TestCoinPricesFromCoingecko(t *testing.T) {
 		"YOUR_COINGECKO_API_KEY",
 	}
 
-	NewProjectTest(t, projectDir).
+	NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
 		CopyFile("tmp/x.wasm").
 		CopyFile("config.toml").
 		RenderTemplateFileFromEnvWithCleanup("fn-call.json", requiredEnvVars).
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
+		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
 func TestErrorHandlingAttestFnCall(t *testing.T) {
 	projectName := "error_handling_attest_fn_call"
 	projectDir := filepath.Join(examplesDir, projectName)
-	NewProjectTest(t, projectDir).
+	NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
 		CopyFile("tmp/x.wasm").
 		CopyFile("config.toml").
 		CopyFile("successFunc.json").
 		CopyFile("errorFunc.json").
 		CopyFile("panicFunc.json").
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
+		Run(filepath.Join(scriptDir, projectName+".txtar"))
+}
+
+func TestErrorHandlingOnChain(t *testing.T) {
+	projectName := "error_handling_on_chain"
+	projectDir := filepath.Join(examplesDir, projectName)
+
+	expectedOutput1 := `Success: true
+	Error: 
+	Value: {"number":42}`
+	expectedOutput2 := `Success: false
+	Error: expected error
+	Value: null`
```

Comment:
error handling on chain runs two tests. These are the expected outputs for each

Reply:
Discussed over vc, it seems that we can leave the definition of "test passed" to the hardhat test, rather than checking for stdout string values